docs/modules/ROOT/pages/static-site-generators.adoc

```diff
@@ -0,0 +1,94 @@
+Site generators have information about the context of page conversion and can translate this information into contextual page attributes, thus providing the ability to show page context in the transformed page itself.
+On the other hand, the template system wrapping the embedded HTML produced by Asciidoctor generally has some concept of variables to display in the template. Usually certain Asciidoc attributes defined in the header are made available as such "page variables".
+
+== Static site generators associated with the Asciidoctor project
+
+=== https://docs.antora.org[Antora]
+
+Antora (written in JavaScript/Node, using Asciidoctor.js) has the distinction of being the only static site generator constructed originally to natively support Asciidoctor.
+It is designed around a processing pipeline with well-defined stages.
+While there are several ways it could be extensible solely through configuration, currently modifying or extending it involves a straightforward code modification of the pipeline and adding a stage or modifying one of the existing stages.
+
+Sources can be in any number of Git repositories or local file system trees.
+These are combined, independent of the source, into components which end up as top-level divisions of the generated site.
+Source trees are laid out in a logical arrangement that well separates concerns and the source arrangement is mapped to the site structure.
+
+Antora supplies Asciidoctor extensions with extensive context information about the file being processed and the set of source files.
+
+AsciiDoc pages transformed to embeddable HTML are wrapped using resources, including templates, from a separately constructed UI bundle.
+Header attributes beginning with `page-` are made available as variables in the template system.
+The UI bundle may contain additional Javascript helper functions that can access variables from the page as well as global context information.
+
+Antora does not support multi-language sites and support is not amenable to implementation through an extension.
+
+Antora does not support blog-like sites natively but support can easily be added through the extension mechanism.
+
+=== xref:jekyll-integration::index.adoc[Jekyll integration]
+
+The Asciidoctor Jekyll integration (Ruby) consists of a Jekyll plugin that uses Asciidoctor to transform source to HTML.
+
+Jekyll supports sources from a single file system tree.
+There is limited location mapping support using a `permalink` page variable.
+
+Layout templates, in the `Liquid` template system, are part of a Jekyll site project.
+Header attributes beginning with `page-` are made available to the template system.
+
+Jekyll does not support multi-language sites in any obvious way.
+
+Jekyll has some native blog support, although this does not affect it's use of Asciidoctor.
+
+=== xref:maven-tools::index.adoc[Maven-tools]
```

Comment:
Do we want to mention maven specifically, maybe better a generinc last section like "Build tools" explainig the shortcomings of those in general?
For name some example content "they help to aggregate and build sets of sources, manage resources(images, etc) locations, but not handle internal links, navigation (menus), etc."

Reply:
Hmmmm.... This was intended to list the SSGs specifically associated with Asciidoctor.  Are there any other build tools that provide an attempt at Asciidoctor integration that integrates with some sort of site building behavior?
How about adding a phrase like "in common with other build tools that can incidentally generate sites,...."? (I haven't tried to fit that in anywhere yet.)

Reply:
I would make the case that the Gradle plugin does as well. It's not a 1-to-1 match with the Maven site generation, but it can be be effectively used for this purpose.

Reply:
I've  investigated the Gradle plugin a bit and added a paragraph.... I'd like the maven tools and gradle plugin maintainers to review in case I've missed something obvious.